### PR TITLE
Bump/0.21 stable

### PR DESCRIPTION
**Pull request by armandfardeau**

### CODE REVIEW

File: .github/run_erblint.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+shopt -s globstar
+
+bundle exec erblint --lint-all
+
+# Store the return code of the erblint execution
+EXIT_CODE=$?
+
+shopt -u globstar
+
+exit $EXIT_CODE
```
Comment (armandfardeau) [inline .github/run_erblint.sh:12]: Please check permissions on file

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+name: Tests
```
Comment (armandfardeau) [inline .github/workflows/tests.yml]: Please use 
```suggestion
name: [CI] Polis
```

File: app/commands/decidim/polis/admin/update_configuration.rb [hunk 3]
```diff
@@ -25,6 +25,7 @@ def call
           return broadcast(:invalid) if form.invalid?
```
Comment (armandfardeau) [inline app/commands/decidim/polis/admin/update_configuration.rb]: Remove useless line return
```suggestion
```

File: app/views/decidim/polis/polis/show.html.erb [hunk 4]
```diff
@@ -57,8 +57,7 @@
           data-auth_needed_to_write="<%= component_settings.social_sign_in_enabled? %>"
           data-xid="<%= current_user.id %>"
           data-x_name="<%= current_user.name %>"
-          data-x_profile_image_url=<%= avatar_url %>
-              style="padding-left:0px;padding-right:0px;"></div>
+          data-x_profile_image_url=<%= avatar_url %>></div>
```
Comment (armandfardeau) [inline app/views/decidim/polis/polis/show.html.erb]: Could you explain what was the problem here?

File: lib/decidim/polis/test/factories.rb [hunk 5]
```diff
@@ -8,6 +8,49 @@
     manifest_name { :polis }
     participatory_space { create(:participatory_process, :with_steps) }
   end
-
   # Add engine factories here
```
Comment (armandfardeau) [inline lib/decidim/polis/test/factories.rb]: Please remove useless comment
```suggestion
```

File: spec/system/polis_spec.rb [hunk 6]
```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "pol.is", type: :system do
+  include_context "with a component"
+
+  let(:site_id) { ENV["POLIS_SITE_ID"] } # to store in local
```
Comment (armandfardeau) [inline spec/system/polis_spec.rb]: Please remove useless comment
```suggestion
  let(:site_id) { ENV["POLIS_SITE_ID"] }
```

File: spec/system/polis_spec.rb [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+  let(:organization) { create(:organization, polis_site_id: site_id) }
+  let(:user) { create :user, :confirmed, organization: organization }
+  # let(:step) { participatory_process.steps.first }
```
Comment (armandfardeau) [inline spec/system/polis_spec.rb]: Please remove useless comment
```suggestion
```

File: spec/system/polis_spec.rb [hunk 8]
```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "pol.is", type: :system do
+  include_context "with a component"
```
Comment (armandfardeau) [inline spec/system/polis_spec.rb:6]: Please use a fixed page_id as defined here: `Decidim::Polis::PolisController#page_id`
```
"#{current_component.id}#{params[:participatory_process_slug]}"
```
in order to not generate polis discussions

File: .github/run_erblint.sh (hunk 1 above)
Comment (Dynnammo) [reply .github/run_erblint.sh:12]: Done : just needed to `git add chmod=+x run_erblint.sh`

**Pull request merged**